L3/CustomViews/TextFields/CardTextField.swift

```diff
@@ -0,0 +1,389 @@
+//
+//  CardTextField.swift
+//  L3
+//
+//  Created by Andrii Kuznietsov on 12.12.2023.
+//
+
+import UIKit
+
+class CardTextField: UIView {
+    
+    enum TextFieldComponents {
+        case border, underline, counter, switcher, placehonder, title, text
+    }
+    
+    private enum Const {
+        static let padding: CGFloat = 10
+        static let textFieldTopPadding: CGFloat = 15
+        static let textFieldHeight: CGFloat = 24
+        static let switcherRightPadding: CGFloat = 12
+        static let bottomBorderTopPadding: CGFloat = 2
+        static let counterLabelTopPadding: CGFloat = 5
+        static let titleLabelTopPadding: CGFloat = -7
+    }
+    
+    private enum State {
+        case error, active, inactive
+        
+        var counerColor: UIColor {
+            switch self {
+            case .error:
+                return .red
+            case .active:
+                return .black
+            case .inactive:
+                return .black
+            }
+        }
+        var titleColor: UIColor {
+            switch self {
```

Comment:
```suggestion
        }
        
        var titleColor: UIColor {
            switch self {
```

to make it more redable